client/src/utils/mountVueComponent.js

```diff
@@ -28,3 +28,11 @@ export const mountVueComponent = (ComponentDefinition) => {
     const component = Vue.extend(ComponentDefinition);
     return (propsData, el) => new component({ store, propsData, el });
 };
+
+export const appendVueComponent = ($el, ComponentDefinition, propsData = {}) => {
+    const container = document.createElement("div");
+    $el.empty().append(container);
```

Comment:
I can edit this in my webpack5 branch, but I am moving away from using jquerified elements for the mounting there, too.  We can do the $.empty() bits easily enough in plain old javascript.